lib/db/DB.js

```diff
@@ -16,19 +19,61 @@ class DB {
   }
 
   define() {
-    this.Order = this.sequelize.define('order', {
-      id: { type: Sequelize.INTEGER, primaryKey: true, autoIncrement: true },
-      peerId: Sequelize.INTEGER,
-      invoice: Sequelize.STRING,
-      quantity: Sequelize.DECIMAL(14, 8),
-      price: Sequelize.DECIMAL(9, 8),
+    // P2P
+
+    this.Peer = this.sequelize.define('peers', {
+      id: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
+      nodeKey: { type: DataTypes.STRING, unique: true },
+      ipv4: DataTypes.STRING(15),
+      port: DataTypes.SMALLINT,
     });
 
-    this.Peer = this.sequelize.define('peer', {
-      id: { type: Sequelize.INTEGER, primaryKey: true, autoIncrement: true },
-      nodeKey: { type: Sequelize.STRING, unique: true },
-      ipv4: Sequelize.STRING(15),
-      port: Sequelize.SMALLINT,
+    // DOB
+
+    this.Currency = this.sequelize.define('currencies', {
```

Comment:
`currency` will be pluralized (I checked to make sure it properly pluralizes words ending in -y)

Reply:
same as above